package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dzcode-io/leblad",
-  "version": "0.0.1",
+  "version": "0.0.2",
```

Comment:
Shouldn't this be a breaking change?

```suggestion
  "version": "1.0.0",
```

Reply:
I bumped the patch version as the library is not that mature, so I thought of releasing v1.0 with all our new changes.
What do you think?

Reply:
In that case you could've gone with '.alpha.xx' but since no one is using it for now :') it's fine 👌🏽